getting-started/singleframe.rst

```diff
@@ -0,0 +1,164 @@
+..
+  Brief:
+  This tutorial is geared towards beginners to data processing with the Science Pipelines.
+  Our goal is to guide the reader through a small data processing project to show what it feels like to use the Science Pipelines.
+  We want this tutorial to be kinetic; instead of getting bogged down in explanations and side-notes, we'll link to other documentation.
+  Don't assume the user has any prior experience with the Pipelines; do assume a working knowledge of astronomy and the command line.
+
+.. _getting-started-tutorial-singleframe:
+
+##############################################################################
+Getting started tutorial part 2: calibrating single frames with Pipeline Tasks
+##############################################################################
+
+In this part of the :ref:`tutorial series <getting-started-tutorial>` you'll process individual raw HSC images in the Butler repository (which you assembled in :doc:`part 1 <data-setup>`) into calibrated exposures.
+We'll use the :command:`singleFrame` pipeline to remove instrumental signatures with dark, bias and flat field calibration images.
+This will also use the reference catalog to establish a preliminary WCS and photometric zeropoint solution.
+
+Pipelines are defined in ``YAML`` files.
```

Comment:
Link to the docs for this: https://pipelines.lsst.io/v/daily/modules/lsst.pipe.base/creating-a-pipeline.html ?